.spacelift/config.yml

```diff
@@ -1,9 +1,11 @@
-# version: 2
-# module_version: "0.0.1"
+version: 2
 
-# tests:
-#   - name: Test infrastructure creation
-#     environment:
-#       TF_VAR_region: eu-west-3
-#       TF_VAR_rds_delete_protection_enabled: false
-#       TF_VAR_s3_retain_on_destroy: false
\ No newline at end of file
+# Note: the module version here is just used to run tests via the Spacelift private module registry.
+# The value can be changed as needed and doesn't need to match the tagged module version.
+module_version: "0.0.1"
+
+tests:
+  - name: Custom VPC
+    project_root: examples/custom-vpc
+    environment:
```

Comment:
TF_VAR_rds_delete_protection_enabled: false
TF_VAR_s3_retain_on_destroy: false
?

Reply:
just in case

Reply:
I'm specifying those variables in the example itself: https://github.com/spacelift-io/terraform-aws-spacelift-selfhosted/pull/16/files#diff-44e7c1118b758c268c06716d530e857514fa0f71f1eb95dfcbe761c86cf1e672R22

Reply:
sorry, right